src/backend/Level0/CHIPBackendLevel0.cc

```diff
@@ -1448,21 +1448,21 @@ void *CHIPContextLevel0::allocateImpl(size_t Size, size_t Alignment,
 
 // CHIPDeviceLevelZero
 // ***********************************************************************
-CHIPDeviceLevel0::CHIPDeviceLevel0(ze_device_handle_t *ZeDev,
-                                   CHIPContextLevel0 *ChipCtx, int Idx)
-    : CHIPDevice(ChipCtx, Idx), ZeDev_(*ZeDev), ZeCtx_(ChipCtx->get()),
-      ZeDeviceProps_() {
-  ZeDeviceProps_.pNext = nullptr;
-  assert(Ctx_ != nullptr);
-}
-CHIPDeviceLevel0::CHIPDeviceLevel0(ze_device_handle_t &&ZeDev,
+CHIPDeviceLevel0::CHIPDeviceLevel0(ze_device_handle_t ZeDev,
                                    CHIPContextLevel0 *ChipCtx, int Idx)
     : CHIPDevice(ChipCtx, Idx), ZeDev_(ZeDev), ZeCtx_(ChipCtx->get()),
       ZeDeviceProps_() {
-  ZeDeviceProps_.pNext = nullptr;
```

Comment:
This needs to remain in place due to how clang initialises structure members.

Reply:
ok, fixed